toolchain/lc-compile/src/check.g

```diff
@@ -1754,8 +1754,6 @@
         Id'Position -> Position
 		Id'Namespace -> Namespace
         (|
-			where(Namespace -> id(_))
-		||
```

Comment:
@runrevmark Note that `module com.livecode.canvas` is the unique and slightly pathological case of "qualified id in definition context".